src/js/utils/ParserUtil.js

```diff
@@ -0,0 +1,29 @@
+module.exports = {
+  combineParsers(parsers = []) {
+    parsers = parsers.filter(
+      function (parser) {
+        return typeof parser === 'function';
```

Comment:
This is just to make sure that we are not executing something which is not a function. We will get an array which should contain functions.